packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -1139,7 +1141,8 @@ export class GarbageCollector implements IGarbageCollector {
             this.inactiveNodeUsed(
                 reason,
                 nodePath,
-                nodeStateTracker,
+                nodeStateTracker.state,
```

Comment:
This task isn't about adding a TombstoneObjectX telemetry. It's about throwing an error when a tombstoned object is revived similar to how error is thrown when a tombstone data store is used or requested.